### ISSUE TITLE
Agenda: Local Time / UTC Time / Timezones?

### ISSUE THREAD

**Issue by javiercasares** [WordPress/wordcamp.org#565]
I have been looking at the agenda of several WordCamps and depending on the time zone you are in it is a bit complex to know what the "real" time of the event is.

In some cases the time is that of the place where it takes place (country, usually) and it is usually indicated as being in "Local Time / UTC-9", in others it indicates that it is in "your local time" which means "my time".

In the first case, everyone "looks for life" to know what time it is local, although it is simple because you have the UTC+something as a reference. The problem comes when you have your own local time because the agenda is reorganized and if there are changes in the day the system goes a bit crazy.

Would it be possible that, in the table, there is a drop-down menu where you can select, for example, "local time of the event", in addition to "your local time" and, already set, a list of "timezones" or "UTC-12 to UTC+12, or simply UTC?

+info (example/problem) at [Slack meta-wordcamp](https://wordpress.slack.com/archives/C08M59V3P/p1599214770064000).

**Comment by iandunn** [comment]
Which events were you looking at other than Colombia? It should be consistent between camps 🤔 

In some ways I like the idea of adding the TZ identifier, but it also seems like we're [moving away from online WordCamps](https://make.wordpress.org/community/2020/07/23/moving-forward-with-online-events/). So when we switch back to a context where camps are local events, I'm not sure it makes sense.

Related: #562, #559

**Comment by iandunn** [comment]
Also related: #492

**Comment by javiercasares** [comment]
I found it in events where UTC-something (something +5 more or less). I checked the Asheville also, and some others... but in general was on the events on the "right side of the map" from UK/France/Spain ;) 

I know the goal is not to have online events (I think we will have them also, in a mix with RL events). Also, for example, in Spain we have 2 timezones, so it's also useful for that.

I think the tickets #562, #559, #492 describe the situation clearly.
